source/wp-content/themes/wporg-developer-2023/src/code-table/view.js

```diff
@@ -0,0 +1,25 @@
+const init = () => {
```

Comment:
This isn't running for me, I guess it's not enqueued yet

Reply:
If you visit: http://localhost:8888/reference/functions/add_settings_section/, does the expanding/collapsing work?

Reply:
Yep, expand/collapse works

Reply:
Hmm, suggests there's a `js` error. I'll take a look.

Reply:
I ran into this running wp 6.1-alpha. Updating WordPress solved it. The automatic adding of the `viewScript` was added in 6.1: https://core.trac.wordpress.org/ticket/56470